docs/developer-guide/etendo-classic/how-to-guides/how-to-create-a-callout-that-extends-from-another-callout.md

```diff
@@ -0,0 +1,189 @@
+---
+tags: 
+  - callout
+  - price lists
+  - net amount calculation
```

Comment:
Tags should refer to the main content, not to examples.

- callout
- extended callout
- Etendo Classic